app/services/trials.js

```diff
@@ -86,6 +86,8 @@ function _processData(trials) {
 
         // Computed values here
         daysAfterCompletion: today.diff(trial.completion_date, 'days'),
+        daysBeforeCompletion: _cleanDate(trial.completion_date)
+          .diff(today, 'days'),
```

Comment:
`today` is a `momentjs` instance already, thus implementing `diff()`. It's not really cleaning, more like "make this a `momentjs` instance so I can put some sugar on the syntax"